### PR TITLE
playing w/ ui elements

### PR DESCRIPTION
**Pull request by hallkliz**

### CODE REVIEW

**Review by dereksgh — APPROVED** [review]

File: main.tscn [hunk 1]
```diff
@@ -47,7 +47,6 @@ polygon = PackedVector2Array(-31, 0, 0, 16, 32, 0, 0, -16)
 [node name="Plot2" type="StaticBody2D" parent="TileMap" groups=["plots"]]
 position = Vector2(64, 128)
 script = ExtResource("4_wgdl0")
-built = true
```
Comment (bombasticSlacks) [inline main.tscn:50]: You remove this for a reason? Or is it a weird holdover?

This is the plots already being built.

Comment (hallkliz) [reply main.tscn:50]: Uh
no not deliberately, I might have fucked something in godot

File: main.tscn [hunk 2]
```diff
@@ -80,7 +79,6 @@ polygon = PackedVector2Array(-31, 0, 0, 16, 32, 0, 0, -16)
 [node name="Plot5" type="StaticBody2D" parent="TileMap" groups=["plots"]]
 position = Vector2(-192, 0)
 script = ExtResource("4_wgdl0")
```
Comment (bombasticSlacks) [inline main.tscn:81]:
```suggestion
script = ExtResource("4_wgdl0")
built = true
```

File: main.tscn [hunk 3]
```diff
@@ -113,7 +111,6 @@ polygon = PackedVector2Array(-31, 0, 0, 16, 32, 0, 0, -16)
 [node name="Plot8" type="StaticBody2D" parent="TileMap" groups=["plots"]]
 position = Vector2(192, -64)
 script = ExtResource("4_wgdl0")
```
Comment (bombasticSlacks) [inline main.tscn:113]:
```suggestion
script = ExtResource("4_wgdl0")
built = true
```

File: main.tscn [hunk 4]
```diff
@@ -47,7 +47,6 @@ polygon = PackedVector2Array(-31, 0, 0, 16, 32, 0, 0, -16)
 [node name="Plot2" type="StaticBody2D" parent="TileMap" groups=["plots"]]
 position = Vector2(64, 128)
 script = ExtResource("4_wgdl0")
```
Comment (bombasticSlacks) [inline main.tscn:49]:
```suggestion
script = ExtResource("4_wgdl0")
built = true
```

**Review by bombasticSlacks — CHANGES_REQUESTED** [review]

**Review by bombasticSlacks — APPROVED** [review]

**Pull request merged**